test/dymaptic.GeoBlazor.Core.Test.Blazor.Shared/Components/WMSLayerTests.razor

```diff
@@ -23,26 +23,30 @@
             Assert.AreEqual(layer?.Id, createdLayer.Id);
             layerViewCreated = true;
         }
-        AddMapRenderFragment(
-            @<SceneView @ref="sceneView"
-                                  class="map-view"
-                                  OnViewRendered="renderHandler"
-                                  OnLayerViewCreate="OnLayerViewCreated">
+
+        AddMapRenderFragment(@<SceneView @ref="sceneView"
+                                         class="map-view"
+                                         OnViewRendered="renderHandler"
+                                         OnLayerViewCreate="OnLayerViewCreated">
                 <Map>
-                    <WMSLayer @ref="layer" Url="https://wms.onmaps.de/?key=8e353de86b08dce9e2c4ca0f0f55c67c"
-                              SpatialReference="new SpatialReference(102100)">
+                    <WMSLayer @ref="layer"
+                              Url="https://ows.terrestris.de/osm/service">
+                        <WMSSublayer Name="OSM-WMS"/>
                     </WMSLayer>
                 </Map>
             </SceneView>);
         await WaitForMapToRender();
         int tries = 100;
+
         while (!layerViewCreated && tries > 0)
         {
             await Task.Delay(100);
             tries--;
         }
```

Comment:
We should probably also create a reusable `WaitForLayerViewCreated` method as well. I'm not sure how easy that would be, and doesn't need to be part of this PR, just thinking about it since you were refactoring the other wait method. But we definitely use this pattern a lot.